dali/test/python/test_utils.py

```diff
@@ -424,7 +431,13 @@ def __next__(self):
                     self.rng.randint(min_s, max_s)
                     for min_s, max_s in zip(self.min_shape, self.max_shape)
                 ]
-            if self.dtype == np.float32:
+            if self.val_range:
+                min_val = self.val_range[0]
+                max_val = self.val_range[1]
+                self.test_data.append(
+                    np.array(self.np_rng.random(shape) * max_val, dtype=self.dtype) - min_val
```

Comment:
I am not sure what's the desired output, but shouldn't it be?
```suggestion
                    np.array(self.np_rng.random(shape) * (max_val - min_val) + min_val, dtype=self.dtype)
```
?

Reply:
It should be, thanks